pyaedt/modeler/cad/Primitives.py

```diff
@@ -275,45 +587,4908 @@ def version(self):
+            "SmallEdgeTol:=",
+            small_edge_tolerance,
+            "SmallFaceAreaTol:=",
+            small_face_area_tolerance,
+            "SpikeTol:=",
+            -1,
+            "GashWidthBound:=",
+            -1,
+            "GashAspectBound:=",
+            -1,
+            "BoundingBoxScaleFactor:=",
+            bounding_box_scale_factor,
+            "RemoveHoles:=",
+            remove_holes,
+            "RemoveChamfers:=",
+            remove_chamfers,
+            "RemoveBlends:=",
+            remove_blends,
+            "HoleRadiusTol:=",
+            hole_radius_tolerance,
+            "ChamferWidthTol:=",
+            chamfer_width_tolerance,
+            "BlendRadiusTol:=",
+            blend_radius_tolerance,
+            "AllowableSurfaceAreaChange:=",
+            allowable_surface_area_change,
+            "AllowableVolumeChange:=",
+            allowable_volume_change,
+        ]
+        self.oeditor.HealObject(selections_args, healing_parameters)
+        return True
+
+    @pyaedt_function_handler()
+    def simplify_objects(
+        self,
+        input_objects_list,
+        simplify_type="Polygon Fit",
+        extrusion_axis="Auto",
+        clean_up=True,
+        allow_splitting=True,
+        separate_bodies=True,
+        clone_body=True,
+        generate_primitive_history=False,
+        interior_points_on_arc=5,
+        length_threshold_percentage=25,
+        create_group_for_new_objects=False,
+    ):
+        """Simplify command to converts complex objects into simpler primitives which are easy to mesh and solve.
+
+        Parameters
+        ----------
+        input_objects_list : str
+            List of object names to simplify.
+        simplify_type : str, optional
+            Simplify type. Default value is ``Polygon Fit``.
+            Available values are ``Polygon Fit`` ``Primitive Fit`` or ``Bounding Box``.
+        extrusion_axis : str, optional
+            Extrusion axis. Default value is ``Auto``.
+            Available values are ``Auto`` ``X``, ``Y`` or ``Z``.
+        clean_up : bool, optional
+            Clean up. The default is ``True``.
+        allow_splitting : bool, optional
+            Allow splitting. The default is ``True``.
+        separate_bodies : bool, optional
+            Separate bodies. The default is ``True``.
+        clone_body : bool, optional
+            Clone body. The default is ``True``.
+        generate_primitive_history : bool, optional
+            Generate primitive history.
+            This option will purge the history for selected objects.
+            The default is ``False``.
+        interior_points_on_arc : float, optional
+            Number points on curve. The default is ``5``.
+        length_threshold_percentage : float, optional
+            Number points on curve. The default is ``25``.
+        create_group_for_new_objects : bool, optional
+            Create group for new objects. The default is ``False``.
+
```

Comment:
```suggestion
        interior_points_on_arc : float, optional
            Number of points on the curve. The default is ``5``.
        length_threshold_percentage : float, optional
            Number of points on the curve. The default is ``25``.
        create_group_for_new_objects : bool, optional
            Whether to create a group for new objects. The default is ``False``.

```
I believe the description for  length_threshold_percentage  is a copy and paste error.

Reply:
```suggestion
        interior_points_on_arc : float, optional
            Number points on curve. The default is ``5``.
        length_threshold_percentage : float, optional
            Length threshold percentage. The default is ``25``.
        create_group_for_new_objects : bool, optional
            Create group for new objects. The default is ``False``.

```